src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -52,567 +51,718 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
-                        listContactNeedAccount.add(c);
-                    }
+        //Creates new Account
+        if (contactsNeedAccounts.size() > 0) {
+            UTIL_Debug.debug('****Number of Contacts that need Accounts created: ' + contactsNeedAccounts.size());
+            insertContactAccount(contactsNeedAccounts, dmlWrapper);
+        }
 
-                    // grab the Accounts that need to have the newly assigned Contact Id added to them if:
-                    //    1. If the contact is connected to an Account
-                    if (c.AccountId != null) {
-                        // contacts are connected to Accounts, make the connection in the other direction
-                        if(c.Account.Primary_Contact__c == null) {
-                            mapAccountIdContactId.put(c.AccountId, c.Id);
-                        }
-                        if (checkHouseholdNaming(c.Account.RecordTypeId)) {
-                            setAccountIdHHToRename.add(c.AccountId);
-                        }
-                    }
-	            }
-
-                // AFTER UPDATE
-                if (triggerAction == TDTM_Runnable.Action.AfterUpdate && !TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.ACCT_IndividualAccounts_TDTM_After_Update)) {
-                    // HH account should be created if:
-	                //   1. The account has been blanked out by the user
-	                //   2. The Account Model is not blank
-	                if (c.AccountId == null && defaultRecTypeID != null) {
-	                    listContactNeedAccount.add(c);
-	                }
-
-                    // if the account (any type) has changed
-                    if (c.AccountId != oldContacts[i].AccountId) {
-                        if (c.AccountId != null) {
-                            listAccountIdHHToUpdate.add(c.AccountId);
-
-                            //Used to Adv to inject oppor logic
-                            mapContactIdAccountIdNewMoveOpps.put(c.Id, c.AccountId);
-
-                            if (checkHouseholdNaming(c.Account.RecordTypeId)) {
-                                setAccountIdHHToRename.add(c.AccountId);
-                                if (oldContacts[i].AccountId != null) {
-                                    setAccountIdHHToRename.add(oldContacts[i].AccountId);
-                                }
-                            }
-                        }else{
-                            if (UTIL_CustomSettingsFacade.getSettings().Automatic_Household_Naming__c == true) {
-                                setAccountIdHHToRename.add(oldContacts[i].AccountId);
-                            }
-                        }
+        if (triggerAction == TDTM_Runnable.Action.AfterInsert) {
+            Map<Id,Id> updateAccPrimaryContact = insertWrapper.accIdConIdToUpdate;
+            Set<Id> accountIdHHToRename = insertWrapper.accountIdToRename;
```

Comment:
This confused me at first. Could we maybe get rid of these and just update the logic below?

(So it would be like `if (insertWrapper.accIdConIdToUpdate.size() > 0) {` etc...